### PR TITLE
feat(@schematics/angular) add .history directory to gitignore

### LINKED ISSUE #12895: Ignore .history directory in git for new angular project
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
```

### Command (mark with an `x`)
```
- [x] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Versions

```bash
v8.11.2
6.4.1

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 7.0.1
Node: 8.11.2
OS: linux x64
Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.10.1
@angular-devkit/core         7.0.1
@angular-devkit/schematics   7.0.1
@schematics/angular          7.0.1
@schematics/update           0.10.1
rxjs                         6.3.3
typescript                   3.1.3
```

### Repro steps
ng n a

### Desired functionality
File `.gitignore` should be created with `.history` entry in the `VSCode` section:

```
# IDE - VSCode
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
.history/*
```

My first step with each project is to write this line, it would be nice to have it already done when creating a new project.

### PR DESCRIPTION
**Pull request by hb-coding**
closes #12895

### CODE REVIEW

**Comment by hb-coding** [comment]
I signed it!

File: packages/schematics/angular/workspace/files/__dot__gitignore [hunk 1]
```diff
@@ -26,7 +26,8 @@ speed-measure-plugin.json
 !.vscode/settings.json
 !.vscode/tasks.json
 !.vscode/launch.json
-!.vscode/extensions.json
+!.vscode/extensions.
```
Comment (alan-agius4) [inline packages/schematics/angular/workspace/files/__dot__gitignore]: Why the extension has been removed?

Comment (hb-coding) [reply packages/schematics/angular/workspace/files/__dot__gitignore]: Not sure, I will add that back.

**Comment by hb-coding** [comment]
Will reopen with fixes

**Comment by hb-coding** [comment]
Re-opened in MR #13002

**Pull request closed without merging**